### DISCUSSION TITLE
Can I have release-plz (or the release-plz github action) generate a changelog but not commit it to the repo?

### DISCUSSION THREAD

**Discussion by codyps** [release-plz/release-plz#1436] [Q&A] [answered]
I'd like to see the changelogs in the release PR and have them included in the tags, but I don't want to have the Changelog checked in to the git repository as a file. Is there a way to configure that?

**Comment by marcoieni** [comment, marked as answer]
Hi!
In theory we could add a configuration option for it.
In practice release-plz has already a lot of configuration opinions and going forward I would like to add them only if what you want to achieve isn't easily doable with github actions [output](https://release-plz.ieni.dev/docs/github/output).

Can you try to implement a flow that looks like:

```
if pr_created:
  uncommit changelog files from pr head branch
  git push
```

If you have success we could add your use case to the docs.
